README.md

```diff
@@ -1 +1,11 @@
-# readme take two
+# Stellarpoint
+
+The StellarPoint Project aims to build a Pose 3D viewer
+which will be beneficial to the Geo’s Pose, 3D, and VPS teams.
+Specifically, GeoPose relies on a 2D visualizer as their
```

Comment:
It is unnecessary to write what the other tool's status, just describe your project.
E.g. What's your project goal, what's the project name meaning, what features it support.